api/applications/views.py

```diff
@@ -50,7 +50,7 @@ class ApplicationList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixi
     view_category = 'applications'
     view_name = 'application-list'
 
-    renderer_classes = [renderers.JSONRenderer]  # Hide from web-browsable API tool
+    renderer_classes = [JSONRendererWithESISupport, JSONAPIRenderer, ]  # Hide from web-browsable API tool
```

Comment:
@abought @samchrisinger @brianjgeiger @pattisdr Decision on removing the renderer_classes which would allow token and applications to appear in DRF browseable API. The (current implementation) alternative is that these urls will work but do not return HTML, just JSON.